eth/stagedsync/stage_bodies.go

```diff
@@ -231,6 +232,8 @@ func BodiesForward(
 					return true, nil
 				}
 
+				metrics.UpdateBlockConsumerBodyDownloadDelay(header.Time, *header.Number, logger)
```

Comment:
should we do `if !firstCycle { call update metric }` here to avoid super large delay observations when syncing from scratch/for the first time (same applies to the other stage metrics)?